src/Encoder.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as E from './Encoder'
+
```

Comment:
I thought the bug was here so tested it. The bug was not, but tests are good, so.